person.rb

```diff
@@ -2,15 +2,17 @@
 require './decorators'
 
 class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
+
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @id = Random.rand(1..1000)
-    @name = name
     @age = age
+    @name = name
     @parent_permission = parent_permission
+    @rentals = []
   end
```

Comment:
- [ ] The request by the previous reviewer to add a method for adding a rental  to create association (Person has many rental ) has not been implemented in your Person class. It can be something similar to this :point_down: 

```
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```